frontend/src/components/GoalForm/__tests__/GoalRttapa.js

```diff
@@ -0,0 +1,51 @@
+import '@testing-library/jest-dom';
+import React from 'react';
+import {
+  render, screen, waitFor,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import GoalRttapa from '../GoalRttapa';
+
+describe('GoalRttapa', () => {
+  const renderGoalRttapa = (goalStatus = 'In Progress', onChange = jest.fn(), initialValue = '') => {
+    render(<GoalRttapa
+      isRttapa="yes"
```

Comment:
if you change the value of the `isRttapa` prop to `asdjkashdkasd`, the tests will still pass. is that expected?

I half expected a test failure somewhere because of the `checked` equality thing here:

```javascript
// src/components/GoalForm/GoalRttapa.js
        <Radio disabled={isLoading} name={inputName} id={`${inputName}-yes`} label="Yes" checked={isRttapa === 'Yes'} onChange={() => onChange('Yes')} />
```